### ISSUE TITLE
Can't enable files_external after upgrade to 8.1

### ISSUE THREAD

**Issue by matzex** [owncloud/core#18536]
After the update to owncloud 8.1 I'm not able to enable the files_external app. I get always a blank page in the browser and the following message in the apache2 error logfile:
<code>[Tue Aug 25 04:12:40 2015] [error] [client XXX] PHP Fatal error:  Access level to OC\\Files\\Storage\\Dropbox::getMetaData() must be public (as in class OC\\Files\\Storage\\Common) in /var/www/XXX/cloud/apps/files_external/lib/dropbox.php on line 324, referer: XXX</code>

Ubuntu 12.04.4, PHP 5.4.44, Apache 2.2.22

**Comment by LukasReschke** [comment]
From which version did you upgrade?

**Comment by matzex** [comment]
8.0.3.4 i think. (This is the last backup of the 8.0.x tree shown in the updater app.)

**Comment by PVince81** [comment]
How did you upgrade ? Did you replace the sources using a tarball, or use a package manager or use the updater app ?

**Comment by Xenopathic** [comment]
That was fixed with #15748, which made it into 8.1... cc @DeepDiver1975

**Comment by matzex** [comment]
I tried to upgrade with the updater app, this failed and afterwards I copied the sources manually from the .zip package. After this I had to disable the files_external app with the command line to access my ownCloud again.

Now I copied the files which were changed in #15748 manually (using the version from the 8.1.1 zip) and files_external works. So I think somehow my update process failed and nothing is wrong in the current release version.
Thank you for your help.
